pages/linux/ip.md

```diff
@@ -4,15 +4,23 @@
 
```

Comment:
Plase the 🍒 of the page, add a link to the man page:
```suggestion
> More information: <https://manpages.debian.org/stretch/iproute2/ip.8.en.html>.

```

Reply:
I'm reluctant about adding this link. This link holds the documentation of the `ip` command _relevant to this specific Debian version_, no more, no less. For a RedHat or an ArchLinux user, it doesn't necessarily contain up to date information, therefore is misleading.

If you are okay with it, I'd just add the `man ip` command as the source for more information, as that is the place that holds all the details available on the user's distribution.

Reply:
:thinking: No, the `map {{command}}` is a well known option, but I didn't find the offical manpage for `ip` with the latest version. Could you find a better manpage link?

Reply:
`ip address` is also a well-known command, we still document it here.

My point is that every URL that leads to _a_ man page can be misleading because it is not related to the version the user uses. I might have an iproute package 3 versions newer than a CentOS user. It doesn't make sense for them to browse what the newest package knows, because they can only work with what's available on their distro.

Therefore I say, _if_ we include any "More info" note here, the best possibility is to refer to the man command. If you don't want the `man` command here because it is trivial, then simply don't include any link here.

Reply:
Hrm. I think we should avoid linking to the man page provided by a specific distribution - unless the command is specific to that distro. linux.die.net is fine for generics, but in this case I'd have to agree with @ecklm I think.